### ISSUE TITLE
Please tag a release

### ISSUE THREAD

**Issue by lh3** [ruanjue/wtdbg2#17]
When you are ready, it would be good to tag a stable release (i.e. create a release in the [release page](https://github.com/ruanjue/wtdbg2/releases)) – this is often a request to my projects as well. You may name it "v2.0" if you feel it is really ready for heavy public uses or "v2.0-rc1" if you are less confident. Up to you. Once you tag a release, I will create a bioconda recipe for wtdbg2. Thanks.

**Comment by mmiladi** [comment]
Hi @lh3 @ruanjue,

great to see updates and this major release!
Just wanted to point out there is actually a recipe for wtdbg-1.2.8, that can be updated. However it is using a commit id which is not ideal. And furthermore it is named 'wtdbg'.
https://github.com/bioconda/bioconda-recipes/blob/bc656c004ca3399595959403ca1870e83ac1e50b/recipes/wtdbg/meta.yaml

Best,
Milad

**Comment by ruanjue** [comment]
Dear Milad,

The significant update was wtpoa-cns, the new consensus tool for wtdbg. How can I help in update it in bioconda?

Best,
Jue

**Comment by lh3** [comment]
I have created a [pull request to bioconda](https://github.com/bioconda/bioconda-recipes/pull/11071). Once the check is done, I will merge it. Then wtdbg2 will be available from bioconda.

To update wtdbg2 in bioconda, you should first request commit permission [here](https://github.com/bioconda/bioconda-recipes/issues/1). Whenever you tag a new release, you can update the "version" and the "sha256" lines in bioconda-recipes/wtdbg2/meta.yaml and send a new pull request to bioconda-recipes. Bioconda will compile your code on their VMs. If it passes tests, you can then merge your own PR.

**Comment by mmiladi** [comment]
I think this would depend on your development plan and scope of  wtdbg1 (https://github.com/ruanjue/wtdbg). In case `wtdbg2` would be the primary/only base for wtdbg development, I would also favor updating the current recipe which has name wtdbg and currently version 1.2.8.

**Comment by mmiladi** [comment]
Ok I see this here https://github.com/bioconda/bioconda-recipes/pull/11072 
Great. Thanks for these developments @lh3 @ruanjue
